Neos.Media/Classes/Domain/ValueObject/Configuration/Label.php

```diff
@@ -0,0 +1,41 @@
\x{1F92A}|\\x{1F92B}|\\x{1F92C}|\\x{1F92D}|\\x{1F92E}|\\x{1F92F}|\\x{1F930}|\\x{1F4B4}|\\x{1F4B3}|\\x{1F4B2}|\\x{1F4B0}|\\x{1F927}|\\x{1F931}|\\x{1F4AF}|\\x{1F4AE}|\\x{1F4AD}|\\x{1F4AC}|\\x{1F4AB}|\\x{1F932}|\\x{1F4AA}|\\x{1F4A9}|\\x{1F4A8}|\\x{1F4A7}|\\x{1F928}|\\x{1F4B5}|\\x{1F933}|\\x{1F4C2}|\\x{1F4C7}|\\x{1F4C6}|\\x{1F920}|\\x{1F921}|\\x{1F922}|\\x{1F923}|\\x{1F924}|\\x{1F925}|\\x{1F4C5}|\\x{1F4C4}|\\x{1F4C3}|\\x{1F4C1}|\\x{1F4B6}|\\x{1F4C0}|\\x{1F4BF}|\\x{1F4BE}|\\x{1F4BD}|\\x{1F4BC}|\\x{1F4BB}|\\x{1F50A}|\\x{1F4BA}|\\x{1F926}|\\x{1F4B9}|\\x{1F4B8}|\\x{1F4B7}|\\x{1F4A6}|\\x{1F4A5}|\\x{1F4C9}|\\x{1F47E}|\\x{1F488}|\\x{1F487}|\\x{1F486}|\\x{1F485}|\\x{1F937}|\\x{1F484}|\\x{1F483}|\\x{1F482}|\\x{1F481}|\\x{1F480}|\\x{1F47F}|\\x{1F47D}|\\x{1F48A}|\\x{1F47C}|\\x{1F47B}|\\x{1F47A}|\\x{1F479}|\\x{1F478}|\\x{1F477}|\\x{1F476}|\\x{1F475}|\\x{1F474}|\\x{1F938}|\\x{1F473}|\\x{1F489}|\\x{1F48B}|\\x{1F4A4}|\\x{1F499}|\\x{1F4A3}|\\x{1F4A2}|\\x{1F4A1}|\\x{1F934}|\\x{1F4A0}|\\x{1F49F}|\\x{1F49E}|\\x{1F49D}|\\x{1F49C}|\\x{1F935}|\\x{1F49A}|\\x{1F498}|\\x{1F48C}|\\x{1F497}|\\x{1F496}|\\x{1F936}|\\x{1F495}|\\x{1F494}|\\x{1F493}|\\x{1F492}|\\x{1F491}|\\x{1F490}|\\x{1F48F}|\\x{1F48E}|\\x{1F48D}|\\x{1F4C8}|\\x{1F49B}|\\x{1F4CA}|\\x{1F4ED}|\\x{1F4F7}|\\x{1F4F6}|\\x{1F4F5}|\\x{1F4F4}|\\x{1F4F3}|\\x{1F4F2}|\\x{1F4F1}|\\x{1F6D0}|\\x{1F6D1}|\\x{1F6D2}|\\x{1F4F0}|\\x{1F4EF}|\\x{1F4EE}|\\x{1F4EC}|\\x{1F6CC}|\\x{1F4EB}|\\x{1F4EA}|\\x{1F6EB}|\\x{1F6EC}|\\x{1F4E9}|\\x{1F4E8}|\\x{1F6F4}|\\x{1F6F6}|\\x{1F6F7}|\\x{1F6F8}|\\x{1F910}|\\x{1F911}|\\x{1F912}|\\x{1F913}|\\x{1F4F8}|\\x{1F4F9}|\\x{1F915}|\\x{1F6BA}|\\x{1F509}|\\x{1F508}|\\x{1F507}|\\x{1F506}|\\x{1F6B6}|\\x{1F505}|\\x{1F504}|\\x{1F503}|\\x{1F502}|\\x{1F501}|\\x{1F6B7}|\\x{1F6B8}|\\x{1F6B9}|\\x{1F6BB}|\\x{1F6C5}|\\x{1F6BC}|\\x{1F6BD}|\\x{1F6BE}|\\x{1F6BF}|\\x{1F6C0}|\\x{1F500}|\\x{1F4FF}|\\x{1F4FC}|\\x{1F4FB}|\\x{1F4FA}|\\x{1F6C1}|\\x{1F6C2}|\\x{1F6C3}|\\x{1F6C4}|\\x{1F914}|\\x{1F6F5}|\\x{1F916}|\\x{1F4D5}|\\x{1F4DF}|\\x{1F91A}|\\x{1F4DE}|\\x{1F4DD}|\\x{1F4DC}|\\x{1F917}|\\x{1F4DA}|\\x{1F91B}|\\x{1F4D9}|\\x{1F4D8}|\\x{1F4D7}|\\x{1F4D6}|\\x{1F91C}|\\x{1F4E1}|\\x{1F4D4}|\\x{1F4D3}|\\x{1F4D2}|\\x{1F4D1}|\\x{1F4D0}|\\x{1F91D}|\\x{1F91E}|\\x{1F4CF}|\\x{1F4CE}|\\x{1F4CD}|\\x{1F4CC}|\\x{1F4CB}|\\x{1F91F}|\\x{1F4E0}|\\x{1F4DB}|\\x{1F4E6}|\\x{1F4E5}|\\x{1F4E2}|\\x{1F401}|\\x{1F4E7}|\\x{1F918}|\\x{1F4E3}|\\x{1F4E4}|\\x{1F919}|\\x{2754}|\\x{2755}|\\x{2753}|\\x{274E}|\\x{274C}|\\x{2757}|\\x{26AA}|\\x{26AB}|\\x{2693}|\\x{26A1}|\\x{2795}|\\x{264F}|\\x{2796}|\\x{25FD}|\\x{231A}|\\x{27B0}|\\x{27BF}|\\x{231B}|\\x{23E9}|\\x{23F3}|\\x{2B1B}|\\x{2B1C}|\\x{2B50}|\\x{2B55}|\\x{23EA}|\\x{23EB}|\\x{23F0}|\\x{2797}|\\x{26BE}|\\x{2728}|\\x{26F5}|\\x{264D}|\\x{264C}|\\x{264B}|\\x{264A}|\\x{2649}|\\x{2648}|\\x{2650}|\\x{2651}|\\x{2652}|\\x{26FA}|\\x{26FD}|\\x{2653}|\\x{2705}|\\x{270A}|\\x{25FE}|\\x{26F3}|\\x{26F2}|\\x{26EA}|\\x{270B}|\\x{26D4}|\\x{23EC}|\\x{2614}|\\x{26CE}|\\x{267F}|\\x{26C5}|\\x{26C4}|\\x{264E}|\\x{26BD}|\\x{2615} -]{1,100}$/u";
```

Comment:
Wouldn't it be easier to define labels without emojis. Or is there a better way to validate the label instead this huge regex.

Reply:
I found it quite useful to use emojis in this case. I took quite some time to find out how to properly check for letters in different languages, including emojis, and this was the best I could find. The alternative would be to only allow latin characters.